sources/VT100Terminal.m

```diff
@@ -269,6 +270,18 @@ - (void)setTermType:(NSString *)termtype
     [delegate_ terminalTypeDidChange];
 }
 
+- (void)setAnswerBackString:(NSString *)s
+{
+    s = [s stringByReplacingEscapedChar:'a' withString:@"\x07"];
```

Comment:
I suggest using `[s stringByExpandingVimSpecialCharacters]` instead. It's a bit more capable and already in use elsewhere.